frontend/tasks/pages.js

```diff
@@ -304,27 +231,12 @@ function generateStaticPageFromMarkdown(pageName, pageParam, markdown, params) {
         <a className="blog-link" href="https://medium.com/firefox-test-pilot" target="_blank" rel="noopener noreferrer">Blog</a>
       </div>
     </header>
-    <div className="layout-wrapper static-page-content">
-      <ReactMarkdown source={ markdown } />
-    </div>
 
-    <footer id="main-footer">
```

Comment:
Huh, I wonder if there's a reason why we had this markup here for the footer rather than just reuse the `<Footer>` component like you've done here? They're slightly different, but I wonder if that's just because we've neglected this file while updating the Footer component

Reply:
Oh hey, yeah, this indeed looks like a fix for #3616

Reply:
Also looks like a fix for #3626